### PR TITLE
Changing to catch Throwable for assertion errors

### PR DESCRIPTION
**Pull request by acron0**
Catch Throwable instead of Exception

Also added `clojure.set` to requires because it was causing some errors in the model library

### CODE REVIEW

File: src/witan/workspace_api/schema.clj [hunk 1]
```diff
@@ -1,5 +1,6 @@
 (ns witan.workspace-api.schema
-  (:require [schema.core :as s]))
+  (:require [clojure.set]
```
Comment (otfrom) [inline src/witan/workspace_api/schema.clj:2]: why clojure.set?

Comment (acron0) [inline src/witan/workspace_api/schema.clj:2]: We've seen this issue in the past and it occurred today - things depending on this library fail to compile it due to `java.lang.ClassNotFoundException: clojure.set` ([example](https://gist.github.com/Eleonore9/e6aa8b2266564c134834382581176eb4))

Comment (otfrom) [inline src/witan/workspace_api/schema.clj:2]: ah, that would be b/c of lines 21 & 23

**Comment by otfrom** [comment]
@acron0 looking again at the Exception hierarchy I think you were right to catch Exception here. Let's change the PR to just fix the clojure.set problem (pls edit the title and description)

**Comment by otfrom** [comment]
:shipit:

**Pull request merged**